test/erc20_gateway/erc20_gateway/deposit.js

```diff
@@ -0,0 +1,145 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+
+const ERC20Gateway = artifacts.require('ERC20Gateway');
+
+const { AccountProvider } = require('../../test_lib/utils.js');
+const HashDepositIntent = require('../../consensus-gateway/consensus_gateway_base/utils.js');
+
+const MessageBusUtils = require('../../message_bus/messagebus_utils');
+const Utils = require('../../test_lib/utils');
+
+contract('ERC20Gateway::deposit', async (accounts) => {
+  const accountProvider = new AccountProvider(accounts);
+  let erc20Gateway;
+  let erc20Cogateway;
+  let stateRootProvider;
+  let setupParam;
+  let valueToken;
+  let depositor;
+  let param;
+
+  beforeEach(async () => {
+    erc20Gateway = await ERC20Gateway.new();
+    erc20Cogateway = accountProvider.get();
+    stateRootProvider = accountProvider.get();
+
+    depositor = accountProvider.get();
+    valueToken = await Utils.deployMockToken(depositor, 200);
+
+    console.log('ValueToken address :-', valueToken.address);
```

Comment:
Remove this console.log and multiple new lines.

Reply:
Removed.